ocaml/parsing/jane_syntax.ml

```diff
@@ -2,6 +2,19 @@ open Asttypes
 open Parsetree
 open Jane_syntax_parsing
 
+(** We carefully regulate which bindings we import from [Language_extension]
+    to ensure that we can import this file into the Jane Street internal
+    repo with no changes.
+*)
+module Language_extension = struct
+  include Language_extension_kernel
+  include (
+    Language_extension
+    : Language_extension_kernel.Language_extension_for_jane_syntax)
+end
+
+module _ = Language_extension
```

Comment:
Why do we need this?

Reply:
This suppresses an "unused module" warning if we don't actually use `Language_extension` (which we currently do, but I don't want that warning to pop up if the use site gets deleted in an unrelated change).

Reply:
Ah, I was confused because we currently do.  Can you add that in a comment, then?  Besides that, LGTM

Reply:
```suggestion
(* Suppress the unused module warning so it's easy to keep around the
   shadowing even if we delete use sites of the module. *)
module _ = Language_extension
```